lib/tortoise311/connection/supervisor.ex

```diff
@@ -16,13 +16,13 @@ defmodule Tortoise311.Connection.Supervisor do
 
   @impl Supervisor
   def init(opts) do
-    children = [
+    base_children = [
       {Inflight, Keyword.take(opts, [:client_id])},
       {Receiver, Keyword.take(opts, [:client_id])},
       {Controller, Keyword.take(opts, [:client_id, :handler])},
-      {Telemetry, Keyword.take(opts, [:client_id])}
     ]
-
+    optional_telemetry = Keyword.get(opts, :enable_telemetry, true) && {Telemetry, Keyword.take(opts, [:client_id])}
```

Comment:
How about this for greater clarity?
```
with_telemetry? = Keyword.get(opts, :enable_telemetry, true)
children = if with_telemetry?, do: base_children ++ [{Telemetry, Keyword.take(opts, [:client_id])}], else: base_children
```

Reply:
I'm fine with this. I'll update and repush